Makefile

```diff
@@ -95,6 +98,9 @@ bundle:
 	rm -rf dist/
 	mkdir -p dist/$(PLUGIN_ID)
 	cp $(MANIFEST_FILE) dist/$(PLUGIN_ID)/
+ifneq ($(wildcard $(ASSESTDIR)/.),)
+	cp -r $(ASSESTDIR) dist/$(PLUGIN_ID)/
```

Comment:
Should this be `ASSESTDIR` --> `ASSETSDIR`?

Reply:
Good catch!